### PR TITLE
Update special-order-numbers.md

### PR DESCRIPTION
**Pull request by ddematheu2**

### CODE REVIEW

**Comment by ProbablePrime** [comment]
I've resolved the conflicts here but we still need a link to the form so ive marked this as waiting for author.

**Comment by ddematheu2** [comment]
Waiting on confirmation from Sabya to post

File: special-order-numbers.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
 # Create a Special Order for Telephone Numbers
 
-Details on how to place special orders for telephone numbers will be available soon.
+Customers who are interested in a special order (ex. acquiring large quantities of numbers) can do so by following the process below. 
+
+*PLEASE NOTE:* You need to have an eligible Azure Subscription and Resource in the US (see link here for eligibility criteria) to be able to purchase numbers. 
+
+You will need to work through the following steps: 
+- Download this form: [Manual / Bulk Number Acquisition Form]()
```
Comment (mikben) [inline special-order-numbers.md:8]: Any way we can provide a web-based form link to make it easier for folks?

Comment (ddematheu2) [reply special-order-numbers.md:8]: Question for Sabya, my understanding is that due to privacy concerns we need to do it this way to comply with support desk.

Comment (ddematheu2) [reply special-order-numbers.md:8]: @dassabya15

Comment (dassabya15) [reply special-order-numbers.md:8]: That's correct - we'll need to follow the official guidelines here @ddematheu2 , @mikben

Comment (dassabya15) [reply special-order-numbers.md:8]: Also, could we add the link to the pdf / word file you helped create?

File: special-order-numbers.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
 # Create a Special Order for Telephone Numbers
 
-Details on how to place special orders for telephone numbers will be available soon.
+Customers who are interested in a special order (ex. acquiring large quantities of numbers) can do so by following the process below. 
+
+*PLEASE NOTE:* You need to have an eligible Azure Subscription and Resource in the US (see link here for eligibility criteria) to be able to purchase numbers.
```
Comment (dassabya15) [inline special-order-numbers.md:5]: We'll need to add the link here for link to eligibility criteria...   https://docs.microsoft.com/en-us/azure/communication-services/concepts/telephony-sms/plan-solution#azure-subscriptions-eligibility

File: special-order-numbers.md (hunk 1 above)
Comment (ddematheu2) [reply special-order-numbers.md:8]: Will add once I push the file live. I need to do a quick rename on it before adding here.

File: special-order-numbers.md (hunk 2 above)
Comment (ddematheu2) [inline special-order-numbers.md:5]:
```suggestion
*PLEASE NOTE:* You need to have an [eligible Azure Subscription and Resource in the US](https://docs.microsoft.com/en-us/azure/communication-services/concepts/telephony-sms/plan-solution#azure-subscriptions-eligibility) to be able to purchase numbers. 
```

File: special-order-numbers.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
 # Create a Special Order for Telephone Numbers
 
-Details on how to place special orders for telephone numbers will be available soon.
+Customers who are interested in a special order (ex. acquiring large quantities of numbers) can do so by following the process below. 
+
+*PLEASE NOTE:* You need to have an [eligible Azure Subscription and Resource in the US](https://docs.microsoft.com/en-us/azure/communication-services/concepts/telephony-sms/plan-solution#azure-subscriptions-eligibility) to be able to purchase numbers.
```
Comment (ddematheu2) [inline special-order-numbers.md:5]:
```suggestion
*PLEASE NOTE:* You need to have an [eligible Azure Subscription and Resource in the US](https://docs.microsoft.com/en-us/azure/communication-services/concepts/telephony-sms/plan-solution#azure-subscriptions-eligibility) to be able to purchase numbers. 
```

**Review by manoskow — APPROVED** [review]

**Pull request merged**